clients/python/README.md

```diff
@@ -25,4 +25,6 @@ Client tests use the `pytest` library. To run them:
 3. Run tests
     ```sh
     pytest clients/python/tests
-    ```
\ No newline at end of file
+    ```
+
+You may also run the provided `run-tests.sh` script which will, generate the client library, install it, and then run the tests.
```

Comment:
why not replace all the steps above with this one command?

Reply:
Fixed both